index.bs

```diff
@@ -985,12 +992,14 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
       <section algorithm>
         <h4 id="check-mime-type-support"><dfn abstract-op noexport lt="Check MIME Type Support|check MIME type support">Check MIME Type Support</dfn></h4>
         <p>
-          To [$check MIME type support$], given a [=MIME type record=] |mimeType|
-          and either a {{MediaEncodingType}} or {{MediaDecodingType}}
-          |encodingOrDecodingType|, perform the following
-          steps. They return <code>supported</code> if
-          the MIME type is [=supported by the user agent=], <code>unsupported</code>
-          if the MIME type is not supported:
+          To [$check MIME type support$], given a [=MIME type record=] |mimeType|,
```

Comment:
As this is a boolean check, why not just return true or false?

Reply:
The result is stored in a value that can have a third state, "unknown."